index.html

```diff
@@ -183,23 +183,30 @@ <h3 class="skill-language">Skills</h3>
       </section>
     </section>
 
-    <section class="formx">
-      <div class="contact-form" id="contact">
+    <section class="contact-form">
+      <div class="" id="contact">
```

Comment:
- [ ] Kindly make sure your `Contact` section has the background image in order to match the provided Figma design. I can see that the image you added is not in the `images` folder. Kindly fix that.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/230599105-a95d83f6-abf2-4f6a-bccd-3c61c41076d9.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/230599156-e0b5b96a-5981-4e4c-8363-287d4bedaa14.png)